src/NServiceBus.AcceptanceTests/EndpointTemplates/Requires.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    static class Requires
+    {
+        static readonly TestSuiteConstraints constraints = new TestSuiteConstraints();
```

Comment:
yes this would also work, but you would have to figure out yourself that you have to create a class with this name. The partial class will show you a build error of not implementing the interface when installing the source package, while this approach would just have a build error about a missing class. I think I'd prefer the current approach, but also I don't have a strong opinion on it.